File: Linguagem Java/1017.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
    Scanner entrada = new Scanner(System.in);
    
    int hora = entrada.nextInt();
    int quilometro = entrada.nextInt();
    
    System.out.printf("%.3f\n",(float)(hora*quilometro)/12);
 
    }
 
}

File: Linguagem GO/1017.go
package main
 
import (
    "fmt"
)
 
func main() {
    var hora, quilometro int
    
    fmt.Scan(&hora)
    fmt.Scan(&quilometro)
    
    fmt.Printf("%.3f\n",(float32)(hora*quilometro)/12);
        
}
